Compile issue with libplist Version 2.3.0

libplist Version 2.3.0 has breaking in function plist_from_memory()
- Breaking:
  * plist_from_memory() gets additional parameter

Now is not possible compile this project with a new version libplist:
Error:
cc -DHAVE_CONFIG_H -I. -I..  -I../include -I../include/ios-webkit-debug-proxy   -I/usr/local/include  -I/usr/local/include  -I/usr/local/include    -g -O2 -Wall -Werror -MT ios_webkit_debug_proxy-idevice_ext.o -MD -MP -MF .deps/ios_webkit_debug_proxy-idevice_ext.Tpo -c -o ios_webkit_debug_proxy-idevice_ext.o `test -f 'idevice_ext.c' || echo './'`idevice_ext.c
idevice_ext.c: In function ‘read_pair_record’:
idevice_ext.c:36:3: error: too few arguments to function ‘plist_from_memory’
   36 |   plist_from_memory(record_data, record_size, pair_record);
      |   ^~~~~~~~~~~~~~~~~
In file included from /usr/local/include/libimobiledevice/libimobiledevice.h:36,
                 from ../include/ios-webkit-debug-proxy/idevice_ext.h:16,
                 from idevice_ext.c:18:
/usr/local/include/plist/plist.h:845:27: note: declared here
  845 |     PLIST_API plist_err_t plist_from_memory(const char *plist_data, uint32_t length, plist_t *plist, plist_format_t *format);
      |                           ^~~~~~~~~~~~~~~~~

--- a/configure.ac
+++ b/configure.ac
@@ -38,6 +38,11 @@ AC_DEFINE_UNQUOTED([LIBIMOBILEDEVICE_VERSION], ["$libimobiledevice_version"], [ 
 AC_DEFINE_UNQUOTED([LIBPLIST_VERSION], ["$libplist_version"], [ ])
 AC_DEFINE_UNQUOTED([LIBUSBMUXD_VERSION], ["$libusbmuxd_version"], [ ])
 
+[libplist_version_major=`expr "$libplist_version" : '\([0-9]*\)'`]
+[libplist_version_minor=`expr "$libplist_version" : '[0-9]*\.\([0-9]*\)'`]
+AC_DEFINE_UNQUOTED([LIBPLIST_VERSION_MAJOR], [$libplist_version_major], [ ])
+AC_DEFINE_UNQUOTED([LIBPLIST_VERSION_MINOR], [$libplist_version_minor], [ ])
+
 # Checks for header files.
 AC_HEADER_STDC
 AC_HEADER_RESOLV

--- a/src/idevice_ext.c
+++ b/src/idevice_ext.c
@@ -33,7 +33,11 @@ int read_pair_record(const char *udid, plist_t *pair_record) {
   }
 
   *pair_record = NULL;
+#if LIBPLIST_VERSION_MAJOR >= 2 && LIBPLIST_VERSION_MINOR >= 3
+  plist_from_memory(record_data, record_size, pair_record, NULL);
+#else
   plist_from_memory(record_data, record_size, pair_record);
+#endif
   free(record_data);
 
   if (!*pair_record) {

